edx_exams/apps/core/email.py

```diff
@@ -44,12 +44,13 @@ def send_attempt_status_email(attempt, escalation_email=None):
         contact_url = f'{settings.LMS_ROOT_URL}/support/contact_us'
         contact_url_text = contact_url
 
-    email_subject = f'Proctored exam {exam.exam_name} for user {attempt.user.username}'
+    email_subject = f'Proctored exam {exam.exam_name} for user {attempt.user.full_name}'
```

Comment:
AFAIK this IDA isn't populating full_name when it creates new User objects since up until this point there's been no use for it. I think you'll need to ensure that happens and possibly migrate that value into existing users.

Reply:
Can you share how data is populated in this IDA?

Reply:
I'm not exactly sure, it's part of the cookie cutter installation for all IDAs. I think it's baked into JWT authentication within https://github.com/openedx/edx-drf-extensions somehow. Although, I imagine this could be an issue for all other IDAs as well depending on if the particular bit of code is looking at JWT details vs the User object from the database. This should be on the JWT but doesn't seem to always exist in the User model.